packages/web/src/components/tooltip/Tooltip.tsx

```diff
@@ -22,13 +21,18 @@ export const Tooltip = ({
   shouldWrapContent = true,
   text = ''
 }: TooltipProps) => {
-  // This is part of the render cycle so that when the theme changes the new
-  // color is applied
-  const themedColor = getCurrentThemeColors()[color]
-
   // Keep track of a hidden state ourselves so we can dismiss the tooltip on click
   const [isHiddenOverride, setIsHiddenOverride] = useState(false)
 
+  const theme = useTheme()
+
+  const themedColor =
+    color === 'primary'
+      ? theme.color.secondary.secondary
+      : color === 'white'
+      ? theme.color.special.white
+      : theme.color.secondary.secondary
```

Comment:
so that one allowed any color under the sun, but i think in this case we just want specific "variants" from what i can tell. but when we get proper designs for tooltip through harmony, lets see what we should do.